data-sources/jumpstart/config.yml

```diff
@@ -0,0 +1,8 @@
+id: jumpstart
+displayName: Jumpstart
+description: |
+  The Jumpstart quickstart is a quick and easy way to see how to get started building a quickstart for partner companies.
+install:
+  primary:
+    link:
+      url: https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/new-relic-quickstarts-overview/
```

Comment:
This URL is broken

Reply:
I think we can replace it with `https://developer.newrelic.com/contribute-to-quickstarts`

Reply:
Lets use this url instead: https://developer.newrelic.com/contribute-to-quickstarts